### DISCUSSION TITLE
In otel-collector-contrib，for skywalking traces，only support receiver but no exporter

### DISCUSSION THREAD

**Discussion by jxplus** [apache/skywalking#11515] [Q&A] [answered]
my needs is want traces nginx to skywalking，two way：
1）skywalking lua agent，i tried is ok;
2）use ngx_otel_module.so。

In method 2)，otel-collector-contrib dot not support  exporter traces to skywalking, 
has plan to support it？

**Comment by wu-sheng** [comment, marked as answer]
This is not about planning, it is impossible. 
SkyWalking has its special protocol based on STAM design.

**Comment by jxplus** [comment]
ok，thanks
